aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -597,6 +598,18 @@ Tensor _sparse_csr_mm(const Tensor& mat1, const Tensor& mat2) {
         0.0,
         1.0);
   }
+  if (mat1.layout() == c10::kStrided &&
+      (mat2.layout() == kSparseCsc || mat2.layout() == kSparseBsc)) {
+    // csc.transpose() -> csr or bsc.transpose() -> bsr
+    //  perform (mat2^T x mat1^T)^T
+    return at::addmm(
+               at::zeros({mat2.size(1), mat1.size(0)}, mat1.options()),
+               mat2.t(),
+               mat1.t(),
+               0.0,
+               1.0)
+        .t();
```

Comment:
See the comment above about replacing `t()` with `transpose(-2, -1)` for batch inputs.